addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15723,7 +15723,7 @@ msgstr ""
 #. Description of settings category "System -> Cache -> Video/Audio/DVD cache - Hard disk" with label #14025
 #: system/settings/settings.xml
 msgctxt "#36400"
-msgid "Enable cache for playback of Video, Audio or DVDs from hard disk."
+msgid "Enable cache for playback of Video, Audio or DVD's from hard disk."
```

Comment:
right you are. but this PR doesnt deal with that. The capitalization is handled in https://github.com/xbmc/xbmc/pull/7080 and I dindt base this PR on that one.